src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/Index.cshtml

```diff
@@ -102,6 +102,16 @@ else
                             <p>Apply for the payment, view applications and change your payment details.</p>
                         </div>
                     }
+
+                    @if (Model.Data.ApprenticeshipEmployerType!= ApprenticeshipEmployerType.Levy 
```

Comment:
Shouldn't this be `==` as we want to show the link to levy payers only?

Reply:
Yes it should